udpa/annotations/migrate.proto

```diff
@@ -8,7 +8,7 @@ package udpa.annotations;
 
 import "google/protobuf/descriptor.proto";
 
-option go_package = "github.com/cncf/xds/go/annotations";
+option go_package = "github.com/cncf/udpa/go/annotations";
```

Comment:
Shouldn't these be `github.com/cncf/xds/go/udpa/annotations`?

Reply:
Yes, good catch. updated